src/DSharp.Mscorlib/Collections/ObjectModel/ReadOnlyCollection.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace System.Collections.ObjectModel
+{
+    //
+    [ScriptIgnoreNamespace]
+    [ScriptImport]
+    [ScriptName("Array")]
```

Comment:
Maybe we can use `nameof(Array)` here instead :D

Reply:
Yes, and no, because it refers to ArrayConstructor in JS 🐙